src/sketchup-stl/exporter.rb

```diff
@@ -269,11 +269,13 @@ def self.do_options
         window_options = {
           :title           => STL.translate('STL Export Options'),
           :preferences_key => PREF_KEY,
-          :height          => 160,
-          :width           => 290,
+          :height          => 170,
+          :width           => 300,
           :modal           => true
         }
         window = SKUI::Window.new(window_options)
+        window.background_color = Sketchup::Color.new(240, 240, 240)
```

Comment:
I wonder why this is needed. Default SKUI is to use system colors. If the user change system color theme this hard coded value might make things look odd...